brave/brave-core
# Monthly statement can now be viewed from Rewards page

Fixes brave/brave-browser#930

UI PR: brave/brave-ui#513

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

1. Start Brave and enable Rewards.
2. Add funds to wallet and perform contributions of all types.
3. Accept Ads and UGP Grants (if available)
4. open Rewards page and click 'View monthly statement for more details'
5. View and verify summary, transactions, monthly contributions, auto-contributions, and one-time tips
--
After merge it may be necessary to bring in a prepared profile that includes previous months. Required files need to be matching `publisher_info_db`, `ledger_state`, and `publisher_state`

Ensure that monthly dropdown operates correctly.

## Note
Currently this PR allows for the current month to be displayed so long as wallet has had activity. Before merge, it is expected that only previous months from the current will be displayed.

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
